test/Sync.t.sol

```diff
@@ -137,95 +129,18 @@ contract SyncTest is Test, Deployers, GasSnapshot {
         router.executeActions(actions, params);
     }
 
-    /// @notice When there is no actual balance in the pool, the ZERO_BALANCE stored in transient reserves should never actually used in calculating the amount paid in settle.
```

Comment:
RIP this test, but i think its a great side effect of this PR that this footgunny behaviour is not possible now but

Reply:
You are right, this test is not worth to have at all anymore, removed